crates/ruff/src/rules/pandas_vet/rules/inplace_argument.rs

```diff
@@ -8,6 +8,29 @@ use crate::rules::pandas_vet::fixes::fix_inplace_argument;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for `inplace=True` in code using the `pandas` library.
+    ///
+    /// ## Why is this bad?
+    /// - `inplace=True` often does not provide a performance benefit. It is
+    /// likely to copy dataframes in the background.
+    /// - It encourages mutation rather than immutable data, which is harder to
+    /// reason about and may cause bugs.
+    /// - It removes the ability to use the method chaining style for `pandas`
+    /// code.
```

Comment:
I should have clarified ... I care in particular about the `ruff rule` output not being longer than 80 chars.

Since the `    /// ` is stripped from that the actual lines here can in fact be 88 chars long, so I think `code.` can be still on the previous line.